File: Linguagem C++/Estrutura Condicional/TrocoVerificado.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    double precoUn, dinheiroRecebido,troco,valorTotal;
    int qnt;

    cout << "Preco unitrio do produto: ";
    cin >> precoUn;
    cout << "Quantidade comprada: ";
    cin >> qnt;
    cout << "Dinheiro recebido: ";
    cin >> dinheiroRecebido;

    valorTotal = precoUn * qnt;

    cout << fixed << setprecision(2);
    if (dinheiroRecebido >= valorTotal){
        troco = dinheiroRecebido - valorTotal;
        cout << "TROCO = " << troco << endl;
    }else {
        troco = valorTotal - dinheiroRecebido;
        cout << "DINHEIRO INSUFICIENTE, FALTAM " << troco << " REAIS" << endl;
    };

    return 0;
}

File: Linguagem C#/Condicional/TrocoVerificado.cs
using System;
using System.Globalization;

namespace TrocoVerificado {
    class Program {
        static void Main(string[] args) {

            CultureInfo CI = CultureInfo.InvariantCulture;

            double precoUn, dinheiroRecebido, troco, valorTotal;
            int qnt;

            Console.Write("Preco unitario do produto: ");
            precoUn = double.Parse(Console.ReadLine(),CI);
            Console.Write("Quantidade comprada: ");
            qnt = int.Parse(Console.ReadLine());
            Console.Write("Dinheiro recebido: ");
            dinheiroRecebido = double.Parse(Console.ReadLine(),CI);

            valorTotal = precoUn * qnt;
            
            if (dinheiroRecebido > valorTotal) {
                troco = dinheiroRecebido - valorTotal;
                Console.WriteLine("TROCO = "+troco.ToString("F2",CI));
            }else {
                troco = valorTotal - dinheiroRecebido;
                Console.WriteLine("DINHEIRO INSUFICIENTE, FALTAM "+troco.ToString("F2",CI)+" REAIS");
            }
        }
    }
}
